apps/launch/common-util/config/wagmi.ts

```diff
@@ -0,0 +1,49 @@
+import { createConfig, http } from 'wagmi';
+import { Chain, gnosis, hardhat, mainnet, polygon } from 'wagmi/chains';
+import { coinbaseWallet, injected, safe, walletConnect } from 'wagmi/connectors';
+
+import { RPC_URLS } from 'common-util/constants/rpcs';
+
+import { virtualGnosis, virtualMainnet, virtualPolygon } from '../../tenderly.config';
+
+const mainnetChain =
+  process.env.NEXT_PUBLIC_IS_CONNECTED_TO_TEST_NET === 'true' ? virtualMainnet : mainnet;
+const gnosisChain =
+  process.env.NEXT_PUBLIC_IS_CONNECTED_TO_TEST_NET === 'true' ? virtualGnosis : gnosis;
+const polygonChain =
+  process.env.NEXT_PUBLIC_IS_CONNECTED_TO_TEST_NET === 'true' ? virtualPolygon : polygon;
+
+export const SUPPORTED_CHAINS: [Chain, ...Chain[]] = [
```

Comment:
shouldn't we have the same list of chains in the `apps/launch/common-util/config/supportedChains.tsx` as in here? it's confusing that they are different